lib/foods.js

```diff
@@ -4,37 +4,92 @@
 const herokuUrl = require("./heroku-url")
 const $ = require("jQuery")
 
+$('#newFoodForm').hide();
+
 function getFoods() {
   //counter = 1;
   $.ajax({
     type: "GET",
     url: herokuUrl() + "api/v1/foods",
     success: function(posts) {
-      posts.forEach(function(foodType){
+      posts.reverse().forEach(function(foodType){
         event.preventDefault;
         let button = `<td><button type='button' class='deleteButton' id='${foodType.id}'>Delete</button></td>`
           let toInsert = `<tr id=${foodType.id}><td>${foodType.name}</td><td>${foodType.calories}</td>${button}</tr>`;
-          $(".foodsTable").append(toInsert);
+        $(".foodsTable").append(toInsert);
       });
       deleteFoodListener();
+      createFoodButtonListener();
+      createFoodFormListener();
     }
   })
 }
 
 
 function deleteFoodListener() {
-$('.deleteButton').on('click', function() {
-  console.log('delete button pressed');
-  //debugger;
-
-$.ajax({
-  type: 'DELETE',
-  url: herokuUrl() + `api/v1/foods/${event.target.id}`,
-  success: $(`#${event.target.id}`).hide()
+  $('.deleteButton').on('click', function() {
+
+    $.ajax({
+      type: 'DELETE',
+      url: herokuUrl() + `api/v1/foods/${event.target.id}`,
+      success: $(`#${event.target.id}`).hide()
+    });
+  });
+};
+
+function createFoodButtonListener() {
+  $('#createButton').on('click', function() {
+    $('#newFoodForm').show();
   });
-});
 };
 
+function createFoodFormListener() {
+  $('#createFoodForm').on('submit', function(event) {
+    event.preventDefault();
+
+    let food = $(event.currentTarget.name).val();
+    let calories = $(event.currentTarget.calories).val();
+
+    errorCheck(food, calories);
+
+    let data = $.post(herokuUrl() + `api/v1/foods?name=${food}&calories=${calories}"`)
+      .then(function(data) {
+        let button = `<td><button type='button' class='deleteButton' id='${data.id}'>Delete</button></td>`
+          let toInsert = `<tr id=${data.id}><td>${food}</td><td>${calories}</td>${button}</tr>`;
+        $(".foodsTable").prepend(toInsert);
+        $('#createFoodForm')[0].reset();
+      });
+  });
+};
```

Comment:
@bretfunk Using `JQuery` instead of `ajax` for this? I like seeing us use different techniques on these requests.